browser/src/Services/Themes/ThemeManager.ts

```diff
@@ -319,6 +484,19 @@ export class ThemeManager {
         return this._themeLoader.getAllThemes()
     }
 
+    public async getVimHighlightColors(tokenColors: IVimHighlight[]) {
```

Comment:
It seems like we're actually setting the token colors here, so we might want to rename this to `setTokenColors` instead.